### DISCUSSION TITLE
2021 State of The Platform

### DISCUSSION THREAD

**Discussion by danirabbit** [elementary/edw#24] [Accepted Talks]
Speaker: @danrabbit

I'd like to open the conference with a high level overview of the AppCenter platform and the major advancements that have occurred over the last year. I'll go over developer tools, new APIs, and share some stats about the growth of our audience. Plus, some commentary on next steps.

If there are specific questions or comments about the state of the platform that you'd like addressed, please comment below!

**Comment by hanaral** [comment]
What are your views on building specific UI tools for elementary app developers. I'm not just talking about Gtk, if and when elementary has enough engineers, would you consider maybe building a UI toolkit from scratch that uses a vala-like (or just Vala) language? I'd love to hear your thoughts, as I sometimes wonder if it would be beneficial to not have to put up with issues that mainstream do not want to stick.

**Reply by danirabbit** [reply]
This seems pretty far outside the scope of what I’m going for here, but in general we always try to work upstream first and collaborate with others. I think it would be a better direction for us to add more engineering talent to GTK than to try to start over from scratch.

**Comment by marbetschar** [comment]
Some of the points below are probably already on your plate - others may not even exist. Just some random ideas crossed my mind and I'd love to hear more about:

- AppCenter for Everyone initiative (current state and outlook)
- elementary Wallet
- Prices per region in AppCenter (allowing an App to be "affordable" by default)
- Recurring payments in AppCenter

**Reply by hanaral** [reply]
Also per region thoughts:
- What developers should do for countries where Stripe doesn't work

**Reply by jalcine** [reply]
> Also per region thoughts:
> 
>     * What developers should do for countries where Stripe doesn't work

This is a big thing for me. I know first-hand that attempting to multiplex payment processors is a _big_ pain in the rear but it does the community a big service when you open up funding bridges this way.
